app/app.go

```diff
@@ -162,7 +166,13 @@ var (
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
+		wasm.ModuleName:                {authtypes.Burner},
 	}
+
+	// // module accounts that are allowed to receive tokens
+	// allowedReceivingModAcc = map[string]bool{
+	// 	distrtypes.ModuleName: true,
+	// }
```

Comment:
does this affect on anything?

Reply:
Not yet, but we need it to fund the community pool. Let me uncomment and finish it up.